### PR TITLE
 issue #6 addinng skills section

### PR DESCRIPTION
**Pull request by aarushisingh2004**
As per the the issue allotted to me, issue  #6 
 I had to make the 
->skills
->testimonials
->faq section 

Out of which, I have completed skills section,  for further sections ill send the pull request again ,please review my pr
The added section of skills look like:-
skill1
skill2

### CODE REVIEW

**Comment by aarushisingh2004** [comment]
#6  issue assigned to me, skills section added please review  @i-am-SnehaChauhan

File: index.html [hunk 1]
```diff
@@ -14,6 +16,10 @@
                     <li><a href="#about">About Me</a></li>
                     <li><a href="#projects">Projects</a></li>
                     <li><a href="#contact">Contact</a></li>
+                    <!-- added the skills and FAQ section -->
+                    <li><a href="#Skills">Skills</a></li>
```
Comment (ayushichoudhary-19) [inline index.html]: Let's maintain consistency. Could we keep the first letter of the ID name in lowercase, as in "#skills" and not "#Skills"? And the same goes for others too.

File: index.html [hunk 2]
```diff
@@ -66,8 +72,77 @@ <h3>Project 3</h3>
             </div>
         </section>
 
+        <!-- Skills section -->
+        <section class="skills">
```
Comment (ayushichoudhary-19) [inline index.html]: For the navbar link to work, we need to add `id="skills"` within this tag. Could you please incorporate that?

**Review by ayushichoudhary-19 — CHANGES_REQUESTED** [review]
Please review the comments added to your committed code and incorporate the changes. Thank you!

**Comment by aarushisingh2004** [comment]
ok I will make the changes soon

**Comment by aarushisingh2004** [comment]
please review @ayushichoudhary-19  I changed it to skill_section  and added skill_section to id , I kept the class name as "skill" only because I was using this class in css at many places , so to avoid mistake, please review , if any more changes are required

**Comment by aarushisingh2004** [comment]
please review @ayushichoudhary-19  , I have merged all conflicts and added changes suggested

**Review by ayushichoudhary-19 — APPROVED** [review]

**Comment by ayushichoudhary-19** [comment]
Great job, @aarushisingh2004! Your pull request looks good and I'll be merging it now!

**Pull request merged**